### PR TITLE
Qt: implement simple find dialog for logs

### PR DESCRIPTION
**Pull request by Megamouse**

### CODE REVIEW

File: rpcs3/rpcs3qt/log_frame.cpp [hunk 1]
```diff
@@ -435,3 +443,23 @@ void log_frame::closeEvent(QCloseEvent *event)
...
+
+	if (event->type() == QEvent::KeyPress)
+	{
+		QKeyEvent* e = (QKeyEvent*)event;
+		if (e->modifiers() == Qt::ControlModifier && e->key() == Qt::Key_F)
+		{
+			if (m_find_dialog && m_find_dialog->isVisible())
+				m_find_dialog->close();
+
+			m_find_dialog.reset(new find_dialog((QTextEdit*)object, this));
```
Comment (VelocityRa) [inline rpcs3/rpcs3qt/log_frame.cpp]: Why not `make_unique`?

File: rpcs3/rpcs3qt/find_dialog.h [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+private:
+	QTextEdit* m_text_edit;
+	QLineEdit* m_find_bar;
+	QPushButton* m_find_next;
+	QPushButton* m_find_last;
+
+private Q_SLOTS:
+	void find_next();
+	void find_last();
+};
```
Comment (VelocityRa) [inline rpcs3/rpcs3qt/find_dialog.h]: Not sure if we care about that, but there's no newline at end of the file.

File: rpcs3/rpcs3qt/find_dialog.cpp [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+find_dialog::~find_dialog()
+{
+}
+
+void find_dialog::find_next()
+{
+	if (!m_text_edit || m_find_bar->text().isEmpty())
+		return;
+
+	//m_text_edit->moveCursor(QTextCursor::Start);
```
Comment (VelocityRa) [inline rpcs3/rpcs3qt/find_dialog.cpp]: Leftover?
Still usually a good idea to document commented out code (mostly the reason why it's not uncommented).

File: rpcs3/rpcs3qt/find_dialog.h (hunk 2 above)
Comment (Megamouse) [reply rpcs3/rpcs3qt/find_dialog.h]: now it should be

**Pull request merged**